console/src/components/Services/Data/utils.js

```diff
@@ -805,16 +813,26 @@ export const cascadeUpQueries = (upQueries = []) =>
   });
 
 export const getDependencyError = (err = {}) => {
+  const dflt = {};
```

Comment:
What is it? Seems like it's not used anywhere except for final return

Reply:
I had some other thoughts when I started writing this :joy:
Changing now